frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -348,37 +348,36 @@ const TableContainer = ({
               {customControl && customControl()}
             </span>
           </div>
-          <div className={`${baseClass}__search`}>
-            {/* Render search bar only if not empty component */}
-            {searchable && !wideSearch && (
-              <>
-                <div
-                  className={`${baseClass}__search-input ${
-                    stackControls ? "stack-table-controls" : ""
-                  }`}
-                  data-tip
-                  data-for="search-tooltip"
-                  data-tip-disable={!searchToolTipText}
-                >
-                  <SearchField
-                    placeholder={inputPlaceHolder}
-                    defaultValue={searchQuery}
-                    onChange={onSearchQueryChange}
-                  />
-                </div>
-                <ReactTooltip
-                  effect="solid"
-                  backgroundColor={COLORS["tooltip-bg"]}
-                  id="search-tooltip"
-                  data-html
-                >
-                  <span className={`tooltip ${baseClass}__tooltip-text`}>
-                    {searchToolTipText}
-                  </span>
-                </ReactTooltip>
-              </>
-            )}
-          </div>
+
+          {/* Render search bar only if not empty component */}
```

Comment:
fixes a gap to the right of the "Add integration" button on the integration table